waspc/packages/deploy/src/providers/fly/deploy/deploy.ts

```diff
@@ -61,11 +61,11 @@ export async function deploy(options: DeployOptions): Promise<void> {
 		const inferredBaseName = getInferredBasenameFromClientToml(tomlFilePaths);
 		const deploymentInfo = createDeploymentInfo(inferredBaseName, undefined, options, tomlFilePaths);
 		await buildWasp();
-		await deployClient(deploymentInfo);
+		await deployClient(deploymentInfo, options.buildClientLocally);
 	}
 }
 
-async function deployServer(deploymentInfo: DeploymentInfo) {
+async function deployServer(deploymentInfo: DeploymentInfo, buildServerLocally: boolean) {
```

Comment:
Nit: I prefer prefer using option objects vs. providing `boolean` params.

```
await deployServer({}, true)
```
vs.
```
await deployServer({}, { buildClientLocally: true })
```

Your code is clear to me because you are using it like this:
```
await deployClient(deploymentInfo, options.buildClientLocally);
```
but I'm just looking out for the future, and making this code easier to extend with additional options if we need to.

Reply:
Good point, can do 👍🏻